iris-backend-service/src/main/java/iris/backend_service/locations/search/db/LocationDAO.java

```diff
@@ -41,14 +39,14 @@
 	@Autowired
 	private LocationRepository locationRepo;
 
-	@Transactional
+	// @Transactional(isolation = Isolation.SERIALIZABLE)
 	public void saveLocations(List<Location> locations) {
 
 		locationRepo.saveAll(locations);
 
-		if (isPostgresActive()) {
-			locationRepo.updateTokens();
-		}
+		// if (isPostgresActive()) {
```

Comment:
I think we should not use comments. Let us remove those lines of code. If we ever need them we could go back in Historie.

Reply:
Okay, you win 😉, I have now dismantled the whole useless class.